windows-gaming-driver/src/config.rs

```diff
@@ -218,7 +218,7 @@ pub struct MachineConfig {
     pub threads: Option<u32>,
 
     // convention: gpu is first
-    pub vfio_slots: Vec<String>,
+    pub vfio_slots: Vec<VfioDevice>,
```

Comment:
Actually, yeah: Please rename this now, this is a great opportunity as you have to rewrite your config anyways to switch to the struct. (We can't justify doing it later)